src/generator/AutoRest.Ruby/TemplateModels/MethodTemplateModel.cs

```diff
@@ -334,6 +334,18 @@ public IEnumerable<ParameterTemplateModel> LocalParameters
         }
 
         /// <summary>
+        /// Get the parameters that are actually method parameters in the order they appear in the method signature
+        /// exclude global parameters and constants
+        /// </summary>
+        public IEnumerable<ParameterTemplateModel> MethodParameters
```

Comment:
Are LocalParameters used as is anywhere? or could we modify LocalParameters method instead of creating a new one? 
Any thoughts on the existing comment in the LocalParameters method at https://github.com/Azure/autorest/pull/1481/files#diff-31d7014dc0acde174bbf87a6e0f80747R328 ? I would expect that comment to not be true anymore (not sure how the condition relates to the comment).